explorer/src/lib/components/blocks-list/BlocksList.svelte

```diff
@@ -45,9 +45,9 @@
   <ListItem
     tooltipText="The average fee paid for the transactions within the block"
   >
-    <svelte:fragment slot="term">average fee paid</svelte:fragment>
+    <svelte:fragment slot="term">average fee paid (dusk)</svelte:fragment>
```

Comment:
I noticed this isn't in the Figma design @laremas

Reply:
This has been removed after a discussion with @laremas